contribute.html

```diff
@@ -14,7 +16,18 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
 <p>The Metal<sup>3</sup> community is an open-source community dedicated to the advancement of the Metal<sup>3</sup> project, a Kubernetes-based solution for managing bare metal infrastructure as code. Leveraging Kubernetes and tools like Ironic, Metal<sup>3</sup> allows users to treat physical machines like virtual machines, simplifying provisioning, management, and scalability of bare metal resources</p>
 
 <p>If you want to learn more about the Metal<sup>3</sup> community or get involved in the project, you can visit their official community page at <a href="/community-resources.html">Community Resources</a></p>
-    
-</main>
+  <br>  
+<header class="mk-main__header">
+<h2 class="mk-heading mk-heading--xl mk-m-border">How to Contribute</h2>
+</header>
+<ul class="mk-social-list">
+<li>Familiarize yourself with the <a href="https://github.com/orgs/metal3-io/repositories">Metal<sup>3</sup> repositories</a> on GitHub.</li>
+<li>Follow the documentation to set up your <a href="/try-it.html">development environment</a>. This will allow you to test your changes locally before submitting them.</li>
+<li>Review the existing issues, feature requests, and ongoing discussions to find areas where you can contribute</li>
```

Comment:
Let's just keep existing issues and feature requests here, and provide the link that lists all issues of the project.

Reply:
Do we have links for feature requests?